radix-engine/src/kernel/kernel_main_facade.rs

```diff
@@ -206,6 +206,10 @@ where
         self.current_frame.actor.clone()
     }
 
+    fn get_current_wasm_memory_consumption(&self) -> usize {
```

Comment:
Ah, looks like I accidentally commited this unneeded file. Pushed an emergency commit to remove it from `develop`.